src/domain/event/eventForm/EventForm.tsx

```diff
@@ -170,4 +184,96 @@ const EventForm = () => {
   );
 };
 
+const ImageSelectedForm: React.FC<{
+  image: string;
+  setFieldValue: (
+    field: string,
+    value: any,
+    shouldValidate?: boolean | undefined
+  ) => void;
+}> = ({ setFieldValue, image }) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <ImagePreview image={image} />
+      <div className={styles.imageSelectRow}>
+        <div className={styles.imageSelect}>
+          <DeleteImageButton
+            onClick={() => {
+              ['image', 'photographer', 'imageAltText'].forEach((field) => {
+                setFieldValue(field, '');
+              });
+            }}
+          />
+        </div>
+        <div>
+          <FormGroup>
+            <Field
+              labelText={t('eventForm.basicInfo.photographer')}
+              name="photographer"
+              component={TextInputField}
+            />
+          </FormGroup>
+          <FormGroup>
+            <Field
+              labelText={t('eventForm.basicInfo.labelImageAltText')}
+              name="imageAltText"
+              helperText={t('eventForm.basicInfo.imageAltTextHelp')}
+              component={TextInputField}
+            />
+          </FormGroup>
+        </div>
+      </div>
+    </>
+  );
+};
+
+const SelectImageForm: React.FC = () => {
+  const { t } = useTranslation();
+  return (
+    <div className={styles.imageSelectRow}>
+      <div className={styles.imageSelect}>
+        <FormGroup>
+          <Field
+            labelText={t('eventForm.basicInfo.labelImage')}
+            name="image"
+            component={ImageInputField}
+          />
+        </FormGroup>
+      </div>
+      <div className={styles.imageInstructionsText}>
+        {t('eventForm.basicInfo.descriptionAddImage')}
+      </div>
+    </div>
+  );
+};
+
+const ImagePreview: React.FC<{ image: string }> = ({ image }) => {
+  const { t } = useTranslation();
+  return (
+    <div className={styles.imagePreview}>
+      <p>{t('eventForm.basicInfo.labelImage')}</p>
+      <img
+        className={styles.eventImage}
+        src={image}
+        alt={t('eventForm.basicInfo.eventImageAltText')}
+      />
+    </div>
+  );
+};
+
+const DeleteImageButton: React.FC<{
```

Comment:
This could more generic component so we could use it in other forms as well. Maybe it could accept label and aria-label as props so it would be easier to re-use. 

Could be renamed e.g. DeleteButton